main.js

```diff
@@ -0,0 +1,488 @@
+const cell5 = new BoardSquare(c5, '', true)
+const cell6 = new BoardSquare(c6, '', true)
+const cell7 = new BoardSquare(c7, '', true)
+const cell8 = new BoardSquare(c8, '', true)
+
+/****  functions **********************************************************************/
+// random number generator
+function randomCell(max, min) {
+  return Math.floor(min + Math.random() * (max - min + 1))
+}
+
+//startTimer =
+function myTimer() {
+  elapsedTime += 1
+  document.getElementById('seconds').innerHTML = elapsedTime
+}
+
+let gameTime = () => {
+  setInterval(myTimer, 1000)
+}
+
+// stop timer
+let stopTimer = () => {
+  clearInterval(gameTime)
+}
+
+//enable board - could be a single variable, do not need individual on-of switches
+let enableBoard = () => {
+  cell0.active = true
+  cell1.active = true
+  cell2.active = true
+  cell3.active = true
+  cell4.active = true
+  cell5.active = true
+  cell6.active = true
+  cell7.active = true
+  cell8.active = true
+}
+
+//disable board - could be a single variable, do not need individual on-of switches
+let disableBoard = () => {
+  cell0.active = false
+  cell1.active = false
+  cell2.active = false
+  cell3.active = false
+  cell4.active = false
+  cell5.active = false
+  cell6.active = false
+  cell7.active = false
+  cell8.active = false
+}
+
+    // Computer now decides it's move - UNSUCCESSFUL
+//let dl5500 = () => {   
+  // computer decides which squares are available to put something in
+       // iterate over my boardSquare objects, find objejts whose owner = ""
+       // build array from these
+  
+  // copmuter now generates a random number
+  // uses this number to pick the nth item in the array
+  //let cellPicker = randomCell('# of items in the array', 0)
+  //computer now places O token in the selected cell
+    
+  
+  // cellPicked = 'cell' + cellPicker
+  // cell0.owner = 'O'
+  // c0.textContent = playerO.token
+
+  // for (let cCell of Object.values(cellsSelectable)) {
+  // if (cCell[0] === "") {
+  //   return cCell[1]
+  // }
+  // }
+
+  //      these items below complete the 'turn' of the computer
+
+  //       winCalc()
+  //       if (winCalc() === true) {
+  //         winner1or2()
+  //       } else {
+  //         turnCounter += 1
+
+
+let reset = () => {  // Intialized game conditions
+  playerX.name = 'Player X'
+  playerX.turn = false
+  playerO.name = 'Player O'
+  playerO.turn = false
+  startUp.setAttribute('disabled', '')// start button not clickable
+  document.getElementById('gameType').style.display = 'block'//show game type choosers
+  document.getElementById('players').style.display = 'none'//hide enter 2 names
+  document.getElementById('player').style.display = 'none'// hide enter 1 name
+  document.getElementById('turnIndicator').style.display = 'none'// hide play display
+  document.getElementById('instructions').style.display = 'block'//show instruction line
+  document.getElementById('timer').style.display = 'none'// hide timer display
```

Comment:
These can be assigned to variables like the elements at the top of your file, that way you can access them using a single variable name instead of a long chain. This could improve your readability and organization.